docs/profile_dependencies.md

```diff
@@ -0,0 +1,157 @@
+---
+title: About InSpec Profile Dependencies
+---
+
+# Profile Dependencies
+
+An InSpec profile can bring in the controls and custom resources from another InSpec profile. Additionally, when inheriting the controls of another profile, a profile can skip or even modify those included controls.
+
+## Defining the Dependencies
+
+Before a profile can use controls from another profile, the to-be-included profile needs to be specified in the including profile’s `inspec.yml` file in the `depends` section. For each profile to be included, a location for the profile from where to be fetched and a name for the profile should be included. For example:
```

Comment:
Before a profile can use controls from another profile, the inherited profile needs to be specified in the including profile’s `inspec.yml` file in the `depends` section.